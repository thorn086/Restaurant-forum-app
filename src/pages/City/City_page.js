import React from 'react'
import NavBar from '../../components/NavBar/NavBar'
import {withRouter} from 'react-router-dom'
import RestForumContext from '../../context'
import CityDis from '../../components/CityDis/CityDis'

class City extends React.Component{
    static defaultProps = {
        match: {
          params: {}
        }
      }
    static contextType = RestForumContext

  

   
    render(){
       
        return(
            <div className="state_page">
                <h2 className="state_title">Choose your City</h2>
                <NavBar />
                <div className='city-nav'>
                    <CityDis />
                </div>
            </div>

        )
    }
}
export default withRouter(City)