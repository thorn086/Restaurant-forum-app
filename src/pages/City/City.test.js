import React from 'react';
import City from './City_page';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('City', () => {
    const city = shallow(<City />);
  
    it('renders properly', () => {
        expect(city).toMatchSnapshot();
    });
});