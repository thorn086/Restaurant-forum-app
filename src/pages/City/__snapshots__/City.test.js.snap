// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City renders properly 1`] = `ShallowWrapper {}`;
